client/src/pages/HomePage.css

```diff
@@ -0,0 +1,130 @@
+@import "../index.css";
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.Homepage {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  gap: 20px;
+  background-image: url("../assets/images/Vector.png");
+  background-position: inherit;
+  background-size: cover;
+  background-repeat: no-repeat;
+  width: 100%;
+  height: 83dvh;
+}
+
+.Lettering {
+  position: absolute;
+  top: 200px;
+  left: 200px;
+  width: 250px;
+}
+
+.Where {
+  background-color: #53a4d7;
+  font-size: 18px;
+  color: white;
+}
+
+.femaleAbuSimbel {
+  height: 450px;
+  width: 600px;
+}
+
+.Vaccation {
+  margin-top: 200px;
+  margin-left: 200px;
+  padding-bottom: 10px;
+}
+
+h2 {
+  font-size: 40px;
+  font-weight: bold;
+  text-align: center;
+}
+
+button {
+  color: white;
+  background-color: #1e3d6a;
+  text-align: center;
+  border: none;
+  border-radius: 5px;
+  height: 25px;
+  font-weight: bold;
+  width: 150px;
+}
+
+h3 {
+  position: absolute;
+  top: 300px;
+  left: 650px;
+
+  color: white;
+  font-size: 20px;
+}
+
+.Caire {
+  font-size: 80px;
+}
+#icons {
```

Comment:
Please stay consisent with selector nomenclature. Either use PascalCase or kebab-case everywhere, but not a mix of both.